actions/prepare_archive_package.yaml

```diff
@@ -0,0 +1,33 @@
+---
+    name: 'prepare_archive_package'
+    runner_type: 'run-local'
+    description: 'Creates a compressed archive of a subset of files inside a runfolder, ready to be sent to PDC with TSM.'
+    enabled: true
+    entry_point: ''
+    parameters:
+        timeout:
```

Comment:
Or do we actually want a "no timeout"? Perhaps it can make sense to instead set it semi-ridiculously high, so that we actually notice an error when it takes "too much time"? If we set it to unlimited then long running executions might be easier to "forget"?